### PR TITLE
Add ENV variables for CA bundle and client cert+key

### PR DESCRIPTION
**Pull request by anirudhbagri**
Fixes #

## Changes

Add ENV variables for CA bundle and client cert+key for zipkin exporter

Please provide a brief description of the changes here. Update the
`CHANGELOG.md` for non-trivial changes. If `CHANGELOG.md` is updated,
also be sure to update `spec-compliance-matrix.md` if necessary.

Related issues #
https://github.com/open-telemetry/opentelemetry-python/pull/2064
https://github.com/open-telemetry/opentelemetry-python/pull/2063

Related [oteps](https://github.com/open-telemetry/oteps) #

### CODE REVIEW

File: specification/sdk-environment-variables.md [hunk 1]
```diff
@@ -131,6 +131,9 @@ See [OpenTelemetry Protocol Exporter Configuration Options](./protocol/exporter.
 | ----------------------------- | -------------------------- | ------------------------------------------------------------------------------------------------------------ |
 | OTEL_EXPORTER_ZIPKIN_ENDPOINT | Endpoint for Zipkin traces | <!-- markdown-link-check-disable --> "http://localhost:9411/api/v2/spans"<!-- markdown-link-check-enable --> |
 | OTEL_EXPORTER_ZIPKIN_TIMEOUT    | Maximum time the Zipkin exporter will wait for each batch export | 10s                                                                                              |
+| OTEL_EXPORTER_ZIPKIN_CA_CERIFICATE    | Path to a CA Bundle | -                                                                                              |
```
Comment (anuraaga) [inline specification/sdk-environment-variables.md:134]:
```suggestion
| OTEL_EXPORTER_ZIPKIN_CERTIFICATE    | The trusted certificate to use when verifying a server's TLS credentials. Should only be used for a secure connection. | -                                                                                              |
```

To follow the format of OTLP's similar flag

File: specification/sdk-environment-variables.md [hunk 2]
```diff
@@ -131,6 +131,9 @@ See [OpenTelemetry Protocol Exporter Configuration Options](./protocol/exporter.
...
+| OTEL_EXPORTER_ZIPKIN_CLIENT_CERIFICATE    | Path to client certificate | -                                                                                              |
```
Comment (anuraaga) [inline specification/sdk-environment-variables.md:135]: I think it's good to mention mutual TLS or mTLS in the description for these two flags.

**Review by anuraaga — COMMENTED** [review]
Should we add mTLS cert / key to OTLP too?

File: specification/sdk-environment-variables.md (hunk 2 above)
Comment (anirudhbagri) [reply specification/sdk-environment-variables.md]: That makes sense. I have updated the description.

**Comment by anirudhbagri** [comment]
> Should we add mTLS cert / key to OTLP too?

I am unsure of how OTLP works. But if it's something we can add, we should definitely add there too.

**Comment by anirudhbagri** [comment]
Reopen this.

File: specification/sdk-environment-variables.md [hunk 3]
```diff
@@ -131,6 +131,9 @@ See [OpenTelemetry Protocol Exporter Configuration Options](./protocol/exporter.
...
 | OTEL_EXPORTER_ZIPKIN_ENDPOINT | Endpoint for Zipkin traces | <!-- markdown-link-check-disable --> "http://localhost:9411/api/v2/spans"<!-- markdown-link-check-enable --> |
 | OTEL_EXPORTER_ZIPKIN_TIMEOUT    | Maximum time the Zipkin exporter will wait for each batch export | 10s                                                                                              |
+| OTEL_EXPORTER_ZIPKIN_CERTIFICATE    | The trusted certificate to use when verifying a server's TLS credentials. Should only be used for a secure connection. | -                                                                                              |
+| OTEL_EXPORTER_ZIPKIN_CLIENT_CERIFICATE    | The client certificate to use when connecting via mTLS | -                                                                                              |
+| OTEL_EXPORTER_ZIPKIN_CLIENT_KEY    | The client's private key to use when connecting via mTLS | -                                                                                              |
```
Comment (Aneurysm9) [inline specification/sdk-environment-variables.md:136]: What are these variables supposed to contain?  Filenames?  PEM?  Base36-encoded DER?

If filenames, what should the files contain?

Comment (tigrannajaryan) [reply specification/sdk-environment-variables.md:136]: +1 to clarify the expected content. In the Collector they are paths to files containing PEM-encoded public or private keys.

**Comment by tigrannajaryan** [comment]
(Argh, Github lost my comments, reposting a shorter version).

It would be nicer to make this env variables uniform for all exporters instead of introducing just for Zipkin. They are uniform in the Collector. Also they are named `ca_file`, `cert_file`, `key_file`, would be nice to also make the names consistent with the Collector.

**Comment by tigrannajaryan** [comment]
OK, apparently Github did not lose my comment. This is a duplicate of https://github.com/open-telemetry/opentelemetry-specification/pull/1968

**Pull request closed without merging**